src/mock/data/dispensers.ts

```diff
@@ -0,0 +1,10 @@
+export default function fakeDispensers(): Record<string, string[]> {
+    /**
```

Comment:
please reformat to 2-space indents

Reply:
Thank for noticing this. Done, this should be solved by husky in the future.